index.js

```diff
@@ -1,78 +1,88 @@
-  bookItem.innerHTML = `
+  static addBookToList(book) {
+    const list = document.getElementById('book-list');
+    // Create tr element
+    const bookItem = document.createElement('div');
+    bookItem.id = `${book.id}`;
+    // Insert columns
+    bookItem.innerHTML = `
     <p>${book.title}</p>
     <p>${book.author}</p>
     <p>${book.isbn}</p>
     <button><a href="#" class="delete">Remove</a></button>
-  `;
-  list.appendChild(bookItem);
-};
-
-// Delete Book
-Display.prototype.deleteBook = function (target) {
-  if (target.className === 'delete') {
-    target.parentElement.parentElement.remove();
+    `;
+    list.appendChild(bookItem);
   }
-};
-
-// Clear Fields
-Display.prototype.clearFields = function () {
-  document.getElementById('title').value = '';
-  document.getElementById('author').value = '';
-  document.getElementById('isbn').value = '';
-};
-
-// Obtain books from local storage on page load
-function getBooks() {
-  let books;
-  if (localStorage.getItem('books') === null) {
-    books = [];
-  } else {
-    books = JSON.parse(localStorage.getItem('books'));
+
+  // Delete Book
+  static deleteBook(target) {
+    if (target.className === 'delete') {
+      target.parentElement.parentElement.remove();
+    }
+  }
+
+  // Clear Fields
+  static clearFields() {
+    document.getElementById('title').value = '';
+    document.getElementById('author').value = '';
+    document.getElementById('isbn').value = '';
   }
-  return books;
-}
-// Display books in local storage
-function displayBooks() {
-  const books = getBooks();
-  books.forEach((book) => {
-    const display = new Display();
-    // Add book to Display
-    display.addBookToList(book);
-  });
-}
-// Add books to local storage
-function addBook(book) {
-  const books = getBooks();
-  books.push(book);
-  localStorage.setItem('books', JSON.stringify(books));
 }
-// Remove books from local storage
-function removeBook(isbn) {
-  let books = getBooks();
-  const filteredBooks = books.filter((book) => book.isbn !== isbn);
-  books = filteredBooks;
-  localStorage.setItem('books', JSON.stringify(books));
+
+// Class local storage
+
+class MylocalStorage {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  // Display books in local storage
+  static displayBooks() {
+    const books = MylocalStorage.getBooks();
+    books.forEach((book) => {
+      // Add book to Display
+      Display.addBookToList(book);
+    });
+  }
+
+  // Add books to local storage
+  static addBook(book) {
+    const books = MylocalStorage.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  // Remove books from local storage
+  static removeBook(isbn) {
+    let books = MylocalStorage.getBooks();
+    const filteredBooks = books.filter((book) => book.isbn !== isbn);
```

Comment:
**Changes Required**
Good effort implementing all the class methods, however;
- [ ] The book currently allows duplicates of `isbn` and when I enter two or more books with the same ISBN, when I try to remove one book, it removes all the books. Let us try to make sure the `isbn` field does not allow duplicates so that we can only remove the exact book.